res/values-pt-rBR/strings.xml

```diff
@@ -451,8 +451,8 @@
 
 
     <string name="Copy_ID">"Copiar ID"</string>
-    <string name="View_Profile">"Ver Perfil"</string>
-    <string name="Send_Message">"Enviar Mensagem"</string>
+    <string name="View_Profile">"Visualizar Perfil"</string>
+    <string name="Send_Message">"Enviar Uma Mensagem"</string>
```

Comment:
Line 454: Previous translation was correct
Line 455: Inserting "uma" will make the string length too long. Previous translation was correct